src/js/components/Tabs/stories/AlignControls.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Attraction, Car, TreeOption } from 'grommet-icons';
+import { Box, Grommet, grommet, Tab, Tabs } from 'grommet';
+import { deepMerge } from 'grommet/utils';
+
+const myTheme = deepMerge(grommet, {
+  tabs: {
+    header: {
+      border: {
+        side: 'bottom',
+        color: 'blue',
+        size: 'small',
+      },
+    },
+  },
+  tab: {
+    border: {
+      side: 'bottom',
+      color: 'dark-4',
+    },
+    pad: 'small',
+    margin: {
+      // bring the overall tabs border behind invidual tab borders
+      vertical: '-2px',
+      horizontal: 'none',
+    },
+  },
+});
+
+const AlignControls = () => (
+  <Grommet theme={myTheme} full>
+    <Tabs justify="start">
+      <Tab title="Tab 1">
+        <Box fill pad="large" align="center">
+          <Attraction size="xlarge" />
+        </Box>
+      </Tab>
+      <Tab title="Tab 2">
+        <Box fill pad="large" align="center">
+          <TreeOption size="xlarge" />
+        </Box>
+      </Tab>
+      <Tab title="Tab 3">
+        <Box fill pad="large" align="center">
+          <Car size="xlarge" />
+        </Box>
+      </Tab>
+    </Tabs>
```

Comment:
I think we can remove this portion of the example, looks awkward 
![image](https://user-images.githubusercontent.com/6320236/83840831-c953db80-a6bc-11ea-9e8a-a0984bf286b1.png)

Reply:
Sounds good. Removed it.